BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -240,6 +240,9 @@ SecStartup2 (
     DEBUG ((DEBUG_INIT, "\n%a\n", mBootloaderName));
   }
 
+  if(Stage1aAsmHob->BistVal != 0)
+    CpuHalt ("BIST failure!\n");
```

Comment:
We can make it more specific by saying "CPU BIST failure!"

Reply:
Ack